pymatgen/io/lobster/inputs.py

```diff
@@ -116,176 +115,139 @@ class Lobsterin(UserDict, MSONable):
         "kpointwiseSpilling",
         "LSODOS",
     )
-    # several of these keywords + ending can be used in a lobsterin file:
-    LISTKEYWORDS = ("basisfunctions", "cohpbetween", "createFatband")
+    # These keywords + ending can be used in a lobsterin file
+    _LIST_KEYWORDS: tuple[str, ...] = (
+        "basisfunctions",
+        "cohpbetween",
+        "createFatband",
+    )
 
-    # all keywords known to this class so far
-    AVAILABLE_KEYWORDS = FLOAT_KEYWORDS + STRING_KEYWORDS + BOOLEAN_KEYWORDS + LISTKEYWORDS
+    # Generate {lowered: original} mappings
+    FLOAT_KEYWORDS: ClassVar = {key.lower(): key for key in _FLOAT_KEYWORDS}
+    STRING_KEYWORDS: ClassVar = {key.lower(): key for key in _STRING_KEYWORDS}
+    BOOLEAN_KEYWORDS: ClassVar = {key.lower(): key for key in _BOOLEAN_KEYWORDS}
+    LIST_KEYWORDS: ClassVar = {key.lower(): key for key in _LIST_KEYWORDS}
 
-    def __init__(self, settingsdict: dict):
+    # All known keywords
+    AVAILABLE_KEYWORDS: ClassVar = {**FLOAT_KEYWORDS, **STRING_KEYWORDS, **BOOLEAN_KEYWORDS, **LIST_KEYWORDS}
+
+    def __init__(self, settingsdict: dict) -> None:
         """
         Args:
             settingsdict: dict to initialize Lobsterin.
         """
         super().__init__()
-        # check for duplicates
-        keys = [key.lower() for key in settingsdict]
+
+        # Check for duplicates from case sensitivity
+        keys = tuple(map(str.lower, settingsdict.keys()))
         if len(keys) != len(set(keys)):
             raise KeyError("There are duplicates for the keywords!")
+
         self.update(settingsdict)
 
-    def __setitem__(self, key, val) -> None:
+    def __setitem__(self, key: str, val: Any) -> None:
         """
-        Add parameter-val pair to Lobsterin. Warns if parameter is not in list of
-        valid lobsterin tags. Also cleans the parameter and val by stripping
-        leading and trailing white spaces. Similar to INCAR class.
-        """
-        # due to the missing case sensitivity of lobster, the following code is necessary
-        new_key = next((key_here for key_here in self if key.strip().lower() == key_here.lower()), key)
-
-        if new_key.lower() not in [element.lower() for element in Lobsterin.AVAILABLE_KEYWORDS]:
-            raise KeyError("Key is currently not available")
-
-        super().__setitem__(new_key, val.strip() if isinstance(val, str) else val)
+        Necessary due to the missing case sensitivity of LOBSTER.
```

Comment:
```suggestion
        Necessary due to the missing case sensitivity of input keywords in LOBSTER.
```

Reply:
Thanks for clarifying this, perhaps it's better for us to use consistent wording here https://github.com/materialsproject/pymatgen/pull/3848#discussion_r1623648797? I would personally prefer "input keywords" over "input commands", because to me, "command" sounds more like something people might type in to the terminal :)

Reply:
"Keyword" and "command" is used interchangeably e.g. `List of keywords`.  🙈 But yeah, I think, although it should be clear what this is referring to, it can't be a bad idea to be language-wise concise. I think `lobsterin keywords` (without the "input" as I suggested above) is the most descriptive and concise.

Reply:
Sure I would change these